src/main/java/org/reaktivity/nukleus/ws/internal/routable/stream/TargetOutputEstablishedStreamFactory.java

```diff
@@ -334,7 +337,12 @@ private void processWindow(
             final int sourceWindowBytesDelta = window.update();
             final int sourceWindowFramesDelta = window.frames();
 
-            final int targetWindowBytesDelta = sourceWindowBytesDelta + targetWindowBytesAdjustment;
+            int targetWindowBytesDelta = sourceWindowBytesDelta + targetWindowBytesAdjustment;
+            if (applyInitialWindowAdjustment)
+            {
+                targetWindowBytesDelta = (int) Math.round((targetWindowBytesDelta * .80));
```

Comment:
`targetWindowBytesAdjustment -= sourceWindowBytesDelta * 20 / 100` in `processInitialWindow`.